### PR TITLE
Add fast generator mode

### PR DESCRIPTION
**Pull request by pakrym**
That reuses existing codemodel.yaml and avoids calling autorest.

### CODE REVIEW

File: eng/Generate.ps1 [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
-param($name, [switch]$noDebug, [switch]$reset)
+param($name, [switch]$noDebug, [switch]$reset, [switch]$noBuild, [switch]$fast)
 
 $ErrorActionPreference = 'Stop'
 
-function Invoke-AutoRest($autoRestArguments)
+function Invoke-AutoRest($baseOutput, $title, $autoRestArguments)
 {
-    $command = "npx autorest-beta $autoRestArguments"
+    $command = "npx autorest-beta $debugFlags $autoRestArguments"
```
Comment (MiYanni) [inline eng/Generate.ps1]: Use $script:debugFlags, since it isn't passed in (explicit scope access to avoid confusion).

File: eng/Generate.ps1 [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
+    {
+        $namespace = $title.Replace('-', '_')
+        $command = "$command --title=$title --namespace=$namespace"
+    }
+
+    if ($fast)
+    {
+        $codeModel = Join-Path $baseOutput $title "CodeModel.yaml"
+        $outputPath = Join-Path $baseOutput $title
+        $command = "dotnet run --project $autorestPluginProject --no-build -- --plugin=cs-modeler --title=$title --namespace=$namespace --standalone --input-file=$codeModel --output-path=$outputPath"
```
Comment (MiYanni) [inline eng/Generate.ps1]: Use $script:autorestPluginProject. And, there is an extra `--` in from of the `--plugin`.

File: src/AutoRest.CSharp.V3/JsonRpc/StandaloneAutoRestInterface.cs [hunk 3]
```diff
@@ -59,6 +59,10 @@ public Task<string[]> ListInputs(string? artifactType = null)
 
         public async Task WriteFile(string filename, string content, string artifactType, RawSourceMap? sourceMap = null)
         {
+            if (string.IsNullOrEmpty(content))
+            {
+                return;
+            }
```
Comment (MiYanni) [inline src/AutoRest.CSharp.V3/JsonRpc/StandaloneAutoRestInterface.cs]: This guard clause should be in the other override (below) too.

File: eng/Generate.ps1 (hunk 2 above)
Comment (pakrym) [reply eng/Generate.ps1]: It's not extra, it's separator for arguments that are for `dotnet run` and the app it's invoking.

Comment (MiYanni) [reply eng/Generate.ps1]: That makes sense.

File: src/AutoRest.CSharp.V3/JsonRpc/StandaloneAutoRestInterface.cs (hunk 3 above)
Comment (pakrym) [reply src/AutoRest.CSharp.V3/JsonRpc/StandaloneAutoRestInterface.cs]: We don't use that one, removed.

File: eng/Generate.ps1 (hunk 1 above)
Comment (pakrym) [reply eng/Generate.ps1]: Done.

**Pull request merged**